enderio-base/src/main/java/com/enderio/base/common/util/GraveCap.java

```diff
@@ -0,0 +1,106 @@
+package com.enderio.base.common.util;
+
+import java.util.Collection;
+import java.util.UUID;
+
+import net.minecraft.core.NonNullList;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.ListTag;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.common.util.Constants;
+
+public class GraveCap implements IGraveCap{
```

Comment:
Now here's the big one:

Maybe we should generalise GraveCap into OwnedInventory (also dropping the prefix of cap)? It makes the actual behaviour and purpose of the capability much clearer and could be reused in future, just a thought. That or have an IOwned capability and then use a regular IItemHandler for storing the items. IOwned would likely be used a lot more than an OwnedInventory. We can speak more about this on Discord.